apps/examples/lcd_test/example_lcd.c

```diff
@@ -234,6 +234,49 @@ static void test_orientation(void)
 	close(fd);
 }
 
+#ifdef CONFIG_LCD_ST7701
+static void test_color_st7701(void)
```

Comment:
Could you explain why we need this? I mean 
why can't we use the existed example?

Reply:
For SPI, the putarea coordinates are : (0, 0) -> (XRES - 1, YRES - 1)
but for MIPI, the putarea coordinates are: (1, 1) -> (XRES, YRES)
So, either we have to change the existing putarea and add separate conditions for both LCD or define a new function separately MIPI based LCD.

Reply:
It's the property of LCDC structure, that the coordinates begin with (1, 1) and end with (480, 800)

Reply:
LCD Driver provides a HAL feature and above is chip-specific. Is there no way to convert above in LCD driver? If not, when we apply new HW, app should be modified. Then it means LCD Driver does not provide a HAL feature.

Reply:
I think this is wrong.

Reply:
yes, I didn't realised that. 
So, instead of doing this in  application part, I have added the changes in st7701.c driver.

Reply:
so, now application wont' be needed to reconfigured differently for mipi and spi.